hs_core/templates/pages/my-resources.html

```diff
@@ -267,6 +268,15 @@ <h4 class="panel-title">
                                     Favorite
                                 {% endif %}
                             </td>
+                            <td>
+                                {% if res.raccess.public %}
+                                    <strong style="color:#5cb85c">&nbspPublic</strong>
+                                {% elif not res.raccess.public and res.raccess.discoverable %}
+                                    <strong style="color:#f0ad4e">&nbspDiscoverable</strong>
+                                {% elif not res.raccess.public and not res.raccess.discoverable %}
+                                    <strong style="color:#d9534f">&nbspPrivate</strong>
+                                {% endif %}
```

Comment:
better to move the style definitions into css so that style definitions can be easily extended/changed and reused for other similar purposes, in this case, showing different sharing status.